Add TALL Stack

driesvints opened the pull request:
<!-- Congrats on creating an Awesome list! 🎉 -->

<!-- Please fill in the below placeholders -->

https://github.com/blade-ui-kit/awesome-tall-stack

I'd like to submit the Awesome TALL Stack list which features a resources on a full stack solution using libraries built by the Laravel community. On here, developers can find resources specific to the TALL stack (and not the individual libraries) to help them learn, code and built their next app.

I hope I did all the steps correctly! 🤞 🙂

sindresorhus commented:
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

driesvints commented:
@sindresorhus urgh, I'm so sorry. I indeed missed some obvious things. 

I reviewed https://github.com/sindresorhus/awesome/pull/1861 & https://github.com/sindresorhus/awesome/pull/1856

Sorry for wasting your time! Hope I got everything right this time.

driesvints commented:
unicorn

JoseDeFreitas commented:
Hello @driesvints 👋.
### Talking about the pull request, I think you have this point wrong:

> This pull request has a title in the format `Add Name of List`.

The pull request title should be "Add Tall Stack" instead of "Add TALL Stack".

### Talking about the list itself, I think you have these points wrong:

> **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)

Your list includes a succinct description, though it's not very descriptive and it's not the same you've written in this repository. It's not written that the succinct description must be the same, but I guess it should so that's why I'm pointing this out.

> The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.

The heading should be "Awesome Tall Stack" instead of "Awesome TALL Stack".

> Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

You list does includes this. Especifically:

> Thanks to the original [awesome project](https://github.com/sindresorhus/awesome) and the Awesome Tailwind CSS repository for inpiring this project.

_My comments are based according to the [guidelines](https://github.com/sindresorhus/awesome/blob/main/pull_request_template.md)._
🤝

JoseDeFreitas reviewed (commented):
Hello 👋.
### Talking about the pull request, I think you have this point wrong:

> This pull request has a title in the format `Add Name of List`.

The pull request title should be "Add Tall Stack" instead of "Add TALL Stack".

### Talking about the list itself, I think you have these points wrong:

> **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)

Your list includes a succinct description, though it's not very descriptive and it's not the same you've written in this repository. It's not written that the succinct description must be the same, but I guess it should so that's why I'm pointing this out.

> The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.

The heading should be "Awesome Tall Stack" instead of "Awesome TALL Stack".

> Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

You list does includes this. Especifically:

> Thanks to the original [awesome project](https://github.com/sindresorhus/awesome) and the Awesome Tailwind CSS repository for inpiring this project.

_My comments are based according to the [guidelines](https://github.com/sindresorhus/awesome/blob/main/pull_request_template.md)._
🤝

guifelix commented:
Ran linter and everything is OK, but I disagree with @JoseDeFreitas about the project name in my humble opinion (this is my first review), he could maintain TALL as it stands for Tailwind, Alpine, Laravel and Livewire; We can see other examples on Awesome list where acronyms are used in capital letters, here are some examples:


- https://github.com/deephacks/awesome-jvm#readme
- https://github.com/sturobson/BEM-resources#readme
- https://github.com/wbkd/awesome-d3#readme

driesvints commented:
@JoseDeFreitas @guifelix thanks for your reviews 👍 

> The pull request title should be "Add Tall Stack" instead of "Add TALL Stack".
> The heading should be "Awesome Tall Stack" instead of "Awesome TALL Stack".

That's the incorrect name.

> Your list includes a succinct description, though it's not very descriptive and it's not the same you've written in this repository. It's not written that the succinct description must be the same, but I guess it should so that's why I'm pointing this out.

Updated.

> You list does includes this. Especifically:

Good one. Removed.

Thanks! : )

danforbes commented:
Very well done! Nice mix of GitHub repos and other types of learning resources like blogs and recordings :tada:

sindresorhus commented on readme.md:
```diff
@@ -296,6 +296,7 @@
 	- [Education](https://github.com/pehapkari/awesome-symfony-education#readme)
 - [Laravel](https://github.com/chiraggude/awesome-laravel#readme) - PHP framework.
 	- [Education](https://github.com/fukuball/Awesome-Laravel-Education#readme)
+	- [TALL Stack](https://github.com/blade-ui-kit/awesome-tall-stack) - Full stack development solution featuring libraries built by the Laravel community.
```
```suggestion
	- [TALL Stack](https://github.com/blade-ui-kit/awesome-tall-stack#readme) - Full-stack development solution featuring libraries built by the Laravel community.
```

sindresorhus commented:
`TALL stack` in `The TALL stack is a full stack development solution featuring some` should be linkified to https://tallstack.dev/

---

In the `The TALL Stack`, add a succinct description to each item.

---

I think it would be clearer if the `Apps & Websites` section was named `Examples`.

driesvints commented:
@sindresorhus all done. Thanks for the review!

driesvints commented:
@sindresorhus the changes you requested seem to give some errors with the awesome list linter: https://github.com/blade-ui-kit/awesome-tall-stack/runs/1341905282

I can't link to tallstack.dev twice. Do you want me to remove the link again or remove the list item?

driesvints commented:
I removed the second link. Should be good now.

sindresorhus commented:
Tweet: https://twitter.com/awesome__re/status/1323617629354008577

driesvints commented:
Thank you so much @sindresorhus! 🥳
